src/List/ItemList.js

```diff
@@ -159,34 +344,20 @@ const Items = props => {
           </NavLink>
         </Fragment>
       ) : (
-        <ul>
-          {dbItems
-            .filter(item =>
-              item.name.toLowerCase().includes(filterInput.toLowerCase())
-            )
-            .map((item, index) => {
-              return (
-                <li key={index}>
-                  <label>
-                    <input
-                      type="checkbox"
-                      name="isPurchased"
-                      checked={is24Hours(item)}
-                      onChange={e => handleChange(e, item)}
-                        value={item.name}
-                    />
-                    {item.name}
-                    <button onClick={e => handleDelete(e)} value={item.name}>
-                      X{" "}
-                    </button>
-                  </label>
-                </li>
-              );
-            })}
-        </ul>
+        <div>
+          {buySoon}
+          {buyKindaSoon}
+          {buyNotSoSoon}
+          {inactive}
+        </div>
       )}
     </div>
   );
 };
 
 export default Items;
+
+// ideas for tomorrow:
```

Comment:
I love comments that express future intent. I think they would be best placed in the area of the code where they are applicable, so they are harder to miss next time you're in here!